### PR TITLE
[thci] unify IPv6 prefix representation

### PR DESCRIPTION
**Pull request by edmont**
Switch to IPv6 dotted-quad format in all uses.

See https://threadgroup.atlassian.net/browse/DEV-2195

### CODE REVIEW

**Comment by edmont** [comment]
FYI @karthick-grl

**Review by karthick-grl — APPROVED** [review]
LGTM

File: tools/harness-thci/OpenThread.py [hunk 1]
```diff
@@ -1894,10 +1858,8 @@ def configExternalRouter(self, P_Prefix, P_stable, R_Preference=0):
             False: fail to configure the border router with a given external route prefix
         """
         print('%s call configExternalRouter' % self)
-        print(P_Prefix)
```
Comment (simonlingoogle) [inline tools/harness-thci/OpenThread.py:1897]: should we assert(isinstance(P_Prefix, str)) to catch potential errors quickly?

File: tools/harness-thci/OpenThread.py [hunk 2]
```diff
@@ -1764,7 +1728,7 @@ def configBorderRouter(
         else:
             pass
 
-        cmd = 'prefix add %s/64 %s %s' % (prefix, parameter, prf)
+        cmd = 'prefix add %s/64 %s %s' % (P_Prefix, parameter, prf)
```
Comment (simonlingoogle) [inline tools/harness-thci/OpenThread.py:1731]: should we assert(isinstance(P_Prefix, str)) to catch potential errors quickly?

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍 

with nits

Is this the change that co-work with fixed TH ?

**Comment by edmont** [comment]
Yes, the change should work with Harness v54.2.

**Comment by simonlingoogle** [comment]
@edmont 
Please fix pretty by running `./script/make-pretty python`

**Pull request merged**